### PR TITLE
fix: columns in ordered items to be delivered report

### PR DESCRIPTION
**Pull request by harshmule-epi**
Asana Task Link:

https://app.asana.com/0/1200471303892628/1200402551564283/f

* Report before changing columns and filters:
Screenshot 2021-06-28 at 10 39 56 AM

* Report after changing columns and filters:
Screenshot 2021-06-28 at 10 38 45 AM

### CODE REVIEW

File: erpnext/stock/report/ordered_items_to_be_delivered/ordered_items_to_be_delivered.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
+frappe.query_reports['Ordered Items To Be Delivered'] = {
+    "filters": [
+        {
+            "fieldname":"customer",
+            "label": __("Customer"),
+            "fieldtype": "Data",
```
Comment (rvpasari) [inline erpnext/stock/report/ordered_items_to_be_delivered/ordered_items_to_be_delivered.js]: change to link field

File: erpnext/stock/report/ordered_items_to_be_delivered/ordered_items_to_be_delivered.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+            "fieldname":"status",
+            "label": __("Status"),
+            "fieldtype": "Select",
+            "width": "80",
+            "options": "\nDraft\nOn Hold\nTo Pick\nTo Pick and Bill\nTo Deliver and Bill\nTo Bill\nTo Deliver\nCompleted\nCancelled\nClosed"
+        },
+        {
+            "fieldname": "item_name",
+            "label": __("Item Name"),
+            "fieldtype": "Data",
```
Comment (rvpasari) [inline erpnext/stock/report/ordered_items_to_be_delivered/ordered_items_to_be_delivered.js]: link field

**Review by rvpasari — CHANGES_REQUESTED** [review]

**Comment by rvpasari** [comment]
@harshmule-git ensure to post a screenshot of how the old report looked like, and how a new report looks like. Also, change the title of the PR to reflect the name of the report

**Comment by sahil28297** [comment]
@harshmule-git why is this in Draft state? Also fix sider issues.

**Comment by harshmule-epi** [comment]
hey @sahil28297 actually working on finding a fix for filter queries which is creating a blocker on the report!

**Comment by hrwX** [comment]
@harshmule-git Whats the status of this PR ?

**Comment by harshmule-epi** [comment]
hey @hrwX we'll be closing this PR after today's discussion, as we have to create a new report!

**Comment by harshmule-epi** [comment]
@rvpasari as discussed, closed this PR!

**Pull request closed without merging**